include/bluetooth/gatt.h

```diff
@@ -280,7 +286,29 @@ struct bt_gatt_cpf {
  *
  * @return 0 in case of success or negative value in case of error.
  */
-int bt_gatt_register(struct bt_gatt_attr *attrs, size_t count);
+#define bt_gatt_register(_attrs, _count)				\
+	bt_gatt_register_service((&(struct bt_gatt_svc) { .attrs = _attrs, \
+				.attr_count = _count, }))
```

Comment:
And since we're on the topic, the name should be bt_gatt_service to match the function name for registration, and the function name should probably be bt_gatt_service_register() to be consistent with Zephyr API naming conventions (think e.g. of bt_le_adv_start() rather than bt_le_start_adv())